README.rst

```diff
@@ -136,15 +136,15 @@ If you use ``anesthetic`` to generate plots for a publication, please cite
 as: ::
 
    Handley, (2019). anesthetic: nested sampling visualisation. Journal of Open
-   Source Software, 4(37), 1414, https://doi.org/10.21105/joss.01414
+   Source Software, 4(37), 1414, https://doi.org/10.2.1.6/joss.01414
 
 or using the BibTeX:
 
 .. code:: bibtex
 
    @article{anesthetic,
-       doi = {10.21105/joss.01414},
-       url = {http://dx.doi.org/10.21105/joss.01414},
+       doi = {10.2.1.6/joss.01414},
+       url = {http://dx.doi.org/10.2.1.6/joss.01414},
```

Comment:
Accidental DOI change?

Reply:
from `bin/bump_version` sed command. Fixed in df622ef

Reply:
fixed in https://github.com/handley-lab/anesthetic/pull/324/commits/247c10c2b905b39e6f6c3749e332ad34a6c54ae2